annotation/chart/templates/deployment.yaml

```diff
@@ -52,27 +73,39 @@ spec:
               name: annotation
               key: POSTGRES_PASSWORD
         - name: POSTGRES_PORT
-          value: "5432"
+          value: "{{ default .Values.global.dbPort .Values.dbPort }}"
         - name: POSTGRES_DB
-          value: "annotation"
+          value: "{{ .Values.dbName }}"
         - name: POSTGRES_HOST
-          value: "postgres-postgresql"
+          value: "{{ default .Values.global.dbHost .Values.dbHost }}"
+        - name: S3_CREDENTIALS_PROVIDER
+          value: "{{ default .Values.global.s3CredentialsProvider .Values.s3CredentialsProvider }}"
         - name: S3_ENDPOINT_URL
-          value: "http://minio"
+          value: "{{ default .Values.global.s3Endpoint .Values.s3Endpoint }}"
+        {{- if .Values.secret.s3user }}
         - name: S3_LOGIN
           valueFrom:
             secretKeyRef:
               name: annotation
               key: S3_LOGIN
+        {{- end }}
+        {{- if .Values.secret.s3password }}
         - name: S3_PASS
           valueFrom:
             secretKeyRef:
               name: annotation
-              key: S3_PASS  
+              key: S3_PASS
+        {{- end }}
+{{- if .Values.extraEnvs }}
```

Comment:
The annotation service has S3_PREFIX property. 
Did you miss that or do we need to configure via extraEnvs?

Reply:
Also. I think extraEnvs should be accessable from global values.